pkgs/development/interpreters/python/hooks/python-imports-check-hook.sh

```diff
@@ -6,8 +6,13 @@ pythonImportsCheckPhase () {
 
     if [ -n "$pythonImportsCheck" ]; then
         echo "Check whether the following modules can be imported: $pythonImportsCheck"
-        export PYTHONPATH="$out/@pythonSitePackages@:$PYTHONPATH"
-        ( cd $out && eval "@pythonCheckInterpreter@ -c 'import os; import importlib; list(map(lambda mod: importlib.import_module(mod), os.environ[\"pythonImportsCheck\"].split()))'" )
+        checkPath=$out
+        if [ -n "$py" ]; then
```

Comment:
Okay. I've added both now. This could theoretically lead to weird behaviour when both are defined (since $python now takes precedence over $py in my implementation), but I think it can be assumed that packages should either not define both, or make libraries importable in both. What do you think?